packages/devtools_app/lib/src/shared/common_widgets.dart

```diff
@@ -1625,28 +1629,28 @@ class CopyToClipboardControl extends StatelessWidget {
 /// [ValueNotifier] and changes to the [ValueNotifier] updating the checkbox.
 class NotifierCheckbox extends StatelessWidget {
   const NotifierCheckbox({
-    Key key,
-    @required this.notifier,
+    Key? key,
+    required this.notifier,
     this.onChanged,
     this.enabled = true,
   }) : super(key: key);
 
   /// The notifier this [NotifierCheckbox] is responsible for listening to and
   /// updating.
-  final ValueNotifier<bool> notifier;
+  final ValueNotifier<bool?> notifier;
 
   /// The callback to be called on change in addition to the notifier changes
   /// handled by this class.
-  final void Function(bool newValue) onChanged;
+  final void Function(bool? newValue)? onChanged;
 
   /// Whether this checkbox should be enabled for interaction.
   final bool enabled;
 
-  void _updateValue(bool value) {
+  void _updateValue(bool? value) {
```

Comment:
All these bools should be able to be non-nullable

Reply:
done

Reply:
nope: CheckBox.onChanged requires handler for nullable value

Reply:
ack